attachment_image_resize/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    attachment_image_resize_models = fields.Char(
+        help="When users attach images to these models, the images will be resized "
+        "based on the maximum resolution specified for attachments."
+    )
+    attachment_image_max_resolution = fields.Char(
+        help="This resolution will be applied to the resizing of images"
+        " for the specified models."
+    )
```

Comment:
Another approach is to have this in ir.model, which makes the size control more granular.

Reply:
It makes sense to me.

Reply:
As we talked, we will use current approach to avoid same configuration for each model and there may be rare case to use each resolution for each model.